appveyor.yml

```diff
@@ -1,33 +1,37 @@
 # version is set in tools\appveyor.psm1 - Invoke-AppVeyorInstall
+environment:
+  POWERSHELL_TELEMETRY_OPTOUT: 1
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
```

Comment:
can we add this link somewhere near here
http://donovanbrown.com/post/Stop-wasting-time-during-NET-Core-builds

Reply:
cc @DarqueWarrior 😄